src/storage/mongodb/BillingStorage.ts

```diff
@@ -234,4 +237,57 @@ export default class BillingStorage {
     // Debug
     await Logging.traceEnd(tenantID, MODULE_NAME, 'deleteInvoice', uniqueTimerID, { id });
   }
+
+  // ------------------------------------------------------------------------------------------
+  // BILLING SETTINGS
+  // ------------------------------------------------------------------------------------------
+  public static async getBillingSetting(tenantID: string): Promise<BillingSettings> {
```

Comment:
Should be in setting's storage class as there is no dedicated storage class for each type of setting.

Reply:
Yes but I don't want to have one storage class per type of integration setting with two methods, this will be overkilling.
What do you think?